apps/landing-page/pages/blog/exploring-quay-registry-in-backstage.mdx

```diff
@@ -0,0 +1,116 @@
+---
+title: Exploring Quay registry in Backstage
+date: 2023/02/20
+description: The Janus IDP family of Backstage plugins is expanding! Please welcome our new member - a frontend plugin that enriches application view in Backstage with insights from Quay registry.
+---
+
+import { Authors } from 'ui/components';
+
+# Exploring Quay registry in Backstage
+
+<Authors authors={['tumido_']} />
+
+The Janus IDP family of Backstage plugins is expanding! Please welcome our new member - a frontend plugin that enriches application view in Backstage with insights from the Quay registry.
+
+Backstage charts and models the software ecosystem as Catalog entities. Users compose small individual service components into a bigger picture. And in order to truly understand and fully describe the individual building blocks, Backstage users construct views to capture different aspects of these components. From technical documentation, through dependencies and relation to deployment state, CI and CD. And part of this picture is understanding what is actually being deployed into live environments: a view of the available and deployed container images. This new Quay plugin for Backstage brings that view.
```

Comment:
Is "charts" somewhat overloading? maybe just drop that adjective as it could be confused with Helm?

Reply:
You may be right, I agree.